src/Microsoft.Crank.Controller/JobSerializer.cs

```diff
@@ -83,7 +96,12 @@ static async Task InitializeDatabaseInternalAsync(string connectionString, strin
             string session,
             string scenario,
             string description,
-            string document
+            string document,
+            bool certBasedAuth = false,
```

Comment:
By "load the cert" do you mean create the `ClientCertificateCredential` at the beginning and pass that around? Even if we load the cert, we still would need to at least pass around the tenant and client id. Admittedly, three things instead of five being passed around is fewer, but it won't be as simple as passing around the cert.